Marina Khokkonen/src/parser.js

```diff
@@ -1,24 +1,82 @@
-const request = require("request");
-const cheerio = require("cheerio");
-request("https://tproger.ru/", (err, response, html)=>{
-    if (err) {
-        console.log("Не удалось получить страницу: ", err);
-    }
-    else if(!err && response.statusCode === 200){
-        const $ = cheerio.load(html),
-            posts = $(".post-text");
-
-        const postsArr = [];
-
-        posts.each(function(i, elem) {
-            postsArr.push({
-                "id" : i,
-                "title" : $(this).find(".entry-title").text(),
-                "content" : $(this).find(".entry-content p").text()
+/*1) Создать на основе express и handlebars веб-сервис с HTML-
+интерфейсом для динамической загрузки информации с одного из
+нескольких сайтов в выбранном формате. Зайдя на этот сервис,
+пользователь сможет с помощью формы настроить параметры
+информационной подборки (например, количество отображаемых
+новостей или их категорию) и получить ее в удобном виде. Форма
+должна отправляться на сервер методом POST.
+)
+2) Реализовать запоминание с помощью cookie текущих настроек
+формы и при заходе на сайт показывать последние использованные
+настройки. Если cookie не существует, можно при отображении
+формы дополнительно учитывать передаваемые GET-запросы
+(например, ?count=10&lang=ru и т.д.)*/
+const path = require('path');
+const request = require('request');
+const cheerio = require('cheerio');
+const consolidate = require('consolidate');
+const express = require('express');
+const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser');
+
+const app = express();
+const urlencodedParser = bodyParser.urlencoded({extended: false});
+
+const tagTypes = [
+                    {tag: 'html', selected: false},
+                    {tag: 'css', selected: false},
+                    {tag: 'javascript', selected: false},
+                    {tag: 'react', selected: false},
+                    {tag: 'node-js', selected: false},
+                 ];
+
+app.set('views', __dirname + '/public');
```

Comment:
Лучше использовать path.resolve для путей
```suggestion
app.set('views', __dirname + '/public');
```
Так же, кажется эта строка - дубль